apps/src/gamelab/actions.js

```diff
@@ -16,6 +16,9 @@ export const CANCEL_LOCATION_SELECTION = 'gamelab/CANCEL_LOCATION_SELECTION';
 export const SELECT_LOCATION = 'gamelab/SELECT_LOCATION';
 export const UPDATE_LOCATION = 'gamelab/UPDATE_LOCATION';
 
+export const CLEAR_CONSOLE = 'spritelab/CLEAR_CONSOLE';
+export const ADD_MESSAGE = 'spritelab/ADD_MESSAGE';
```

Comment:
I'm double checking with Ryan on a couple of these points: I decided to namespace to spritelab because we aren't planning to have the text console in gamelab and because we are eventually planning to split up spritelab and gamelab. I figured this namespacing would make the eventual splitting easier.

Reply:
^ Future plan confirmed.